charts/openfga/templates/job.yaml

```diff
@@ -51,6 +51,11 @@ spec:
                   name: "{{ .Values.datastore.uriSecret }}"
                   key: "uri"
             {{- end }}
+
+            {{- with .Values.extraEnvVars }}
+              {{- toYaml . | nindent 12 }}
+            {{- end }}
```

Comment:
Same as we do for deployment:

https://github.com/openfga/helm-charts/blob/40ea4e1936efb04da540d1d2afe7e6375eedda3f/charts/openfga/templates/deployment.yaml#L326